utilities/similarity/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM nvidia/cuda:9.0-devel-ubuntu16.04 as cuda_devel
+
+FROM tensorflow/serving:1.11.0-gpu
+
+COPY --from=cuda_devel \
+     /usr/local/cuda/lib64/stubs/libcuda.so /usr/local/cuda/lib64/stubs/
+
+# Fix to support running without nvidia-docker.
+RUN ln -s /usr/local/cuda/lib64/stubs/libcuda.so /usr/local/cuda/lib64/stubs/libcuda.so.1 \
+    && echo "/usr/local/cuda/lib64/stubs" > /etc/ld.so.conf.d/z-cuda-stubs.conf \
+    && ldconfig
+
+WORKDIR /root
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+        python-pip \
```

Comment:
`pip` and `setuptools` should be available in the TensorFlow image recommended above.